core/src/test/java/com/datastax/oss/driver/internal/core/metadata/DefaultTopologyMonitorTest.java

```diff
@@ -256,16 +260,17 @@ public void should_get_new_node_from_peers() {
     verify(peer2).getInetAddress("rpc_address");
     verify(peer2, never()).getString(anyString());
 
-    verify(peer1).getInetAddress("rpc_address");
+    // the added node will have its rpc_address tried twice
+    verify(peer1, times(2)).getInetAddress("rpc_address");
```

Comment:
This is new, we cannot reuse the retrieved address anymore inside `buildNodeInfo`, so the address gets computed twice.